x-pack/elastic-agent/pkg/remote/client.go

```diff
@@ -25,7 +25,7 @@ import (
 const (
 	defaultPort = 8220
 
-	retryOnBadConnTimeout = 5 * time.Minute
+	retryOnBadConnTimeout = 6 * time.Minute
```

Comment:
Why this was changed as part of this PR?

Reply:
Yeah why this change? This is not related to long-polling, if your thinking it needs to be larger than 5 minutes from the Fleet Server side.

Reply:
Argh sorry, this should not have been in this PR, I played around with some numbers. Will revert.